Python/Product/Common/Infrastructure/ObservableCollectionExtensions.cs

```diff
@@ -60,5 +60,50 @@ IComparer<T> compareSortKey
                 left.Insert(index, item);
             }
         }
+
+        public static void Merge<TLeft, TRight, TKey>(
```

Comment:
there's a couple of copies of this file, don't know if you want to keep them in sync or not.